src/mavsdk_server/src/CMakeLists.txt

```diff
@@ -112,6 +112,9 @@ if(NOT IOS AND NOT ANDROID)
         set_target_properties(mavsdk_server_bin PROPERTIES
             OUTPUT_NAME mavsdk_server
         )
+
+        # Fix undefined reference to `dlsym' on Linux
+        set(CMAKE_EXE_LINKER_FLAGS "-Wl,--no-as-needed -ldl")
```

Comment:
Hmm feels weird to have to manually set the flag like this :thinking:

Reply:
I know. I tried using `target_link_options` but I couldn't get it right.

Reply:
Is it not just a `target_link_libraries` with `dl`?

Reply:
Hmm. What's `dl`? And why would we need it?

Reply:
Hmm, dynamic linking, but where do we do that?

Reply:
Ok, I think I was able to do it using your suggestion.